GVRf/Framework/src/org/gearvrf/GVRActivity.java

```diff
@@ -136,6 +137,24 @@ public void setScript(GVRScript gvrScript, String distortionDataFileName) {
         }
     }
 
+    /**
+     * Sets whether to force rendering to be single-eye, monoscopic view.
+     *
```

Comment:
I think we may need to either add JavaDoc here to specify this setForceMonoscopic only takes effect before you call setScript or throws an exception like UnsupportedOperationException if someone calls this after setScript().